src/dataflow/utilities/aws.ts

```diff
@@ -75,9 +75,11 @@ export function uploadProgram(programData: NodeEditor): string {
   return "completed";
 }
 
-export const fetchProgramData = (programId: string, time?: number) => {
-
-  const queryParams = { programId, time };
+export const fetchProgramData = (programId: string, time?: number, endTime?: number) => {
```

Comment:
I wonder if there will be any actual need for the end time. I can see leaving the code in, but I don' see when it would even need to be used.

Reply:
This was pulled from Chris's changes here so I'm not sure if perhaps she was just using this as a tool for testing:
https://github.com/concord-consortium/collaborative-learning/pull/369
Maybe there could be some need to request a subset of the data?  I agree, I can't immediately think of a good use for an end time.